services/ows_refactored/surface_reflectance/style_sr_cfg.py

```diff
@@ -775,7 +775,7 @@
         "green": {"green": 1.0},
         "blue": {"blue": 1.0}
     },
-    "scale_range": [1.0, 19636.0],
+    "scale_range": [7274.0, 26909.0],
```

Comment:
I think your original upper bound value may already have included the 7273 offset.

Reply:
shall we just use that [7272.0, 18181.0]?